research-software-directory/RSD-as-a-service
# Fix and cleanup rsd-v2-release

# Fix 3 items and clean unused imports and files

Changes proposed in this pull request:
* Fix: duplicated project entries in organisation project overview
* Fix: decode uri search param when used in the component
* Fix: software and project count for maintainers (incl. rsd_admin)
* Chore: clean unused import and files using `unimported` library, and add it as devDependency
* Feat: add order on project title
* Feat: default links to software and project overview pages include order on mention and impact item count

How to test:
* `make start` to build app and generate data
* Validate no duplicated project entries in an organisation:
   * create few research units under specific organisation
   * add these research units to a project of the parent organisation
   * add parent organisation as funding organisation too
   * confirm that project entry is shown only once in the organisation project list
* Try search for software and projects withing organisation. Use spaces and some other strange charaters. It should work as expected
* Login as rsd_admin. Compare software and project counts in the organisation overview and when you open the organisation page. The counts should be different when there are unpublished and blocked items. The count in the tabl should be same as the count shown on the page (see image)

## Improved project counts in the tab

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/789a35fa-8f4f-460b-9f25-a9daa93774b5)
 
PR Checklist:
